nise/yaml_generators/oci/generator.py

```diff
@@ -25,40 +25,30 @@
 from dateutil.relativedelta import relativedelta
 from nise.util import LOG
 from nise.yaml_generators.generator import Generator
+from nise.yaml_generators.oci.oci_yaml_constants import get_tag_keys
```

Comment:
After thinking about the relationship between `OCITags` and `OCIYamlConstants` more, I don't think it makes sense to couple these objects. The functionality of `get_tag_keys` can be incorporated into the `OCITags` object itself.
```suggestion
from nise.yaml_generators.oci.oci_yaml_constants import OCITags
```